dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -486,15 +503,20 @@ def runCrossValidation(self, jobId, interfaces):
             failures = Validator.crossValidateSql(comboRules.all(),submissionId)
             # get error file name
             reportFilename = self.getFileName(interfaces.errorDb.getCrossReportName(submissionId, row.first_file_name, row.second_file_name))
+            warningReportFilename = self.getFileName(interfaces.errorDb.getCrossWarningReportName(submissionId, row.first_file_name, row.second_file_name))
```

Comment:
I know you didn't change the actual `getFileName` code (beginning on line 157), but I noticed that we might want to make that function more robust by using the `os.path.join` function to ensure we build the file paths correctly across OSs, like we do in other places.